pkg/apis/policy/v1alpha1/well_known_constants.go

```diff
@@ -1,6 +1,12 @@
 package v1alpha1
 
 const (
+	// PropagationPolicyIDLabel is the id of PropagationPolicy object.
+	PropagationPolicyIDLabel = "propagationpolicy.karmada.io/id"
```

Comment:
```suggestion
	// PropagationPolicyPermanentIDLabel is used to reference a PropagationPolicy object permanently.
	// Compared to the uuid in metadata, it can keep unchanged even in case of data backup and
	// recovery by tooling like Velero.
	PropagationPolicyPermanentIDLabel = "propagationpolicy.karmada.io/permanent-id"
```
I suggest giving it a more meaningful name and explaining the difference between the UUID in metata.

Reply:
+1

Reply:
Seems the latest commit didn't change the key name.
> PropagationPolicyPermanentIDLabel = "propagationpolicy.karmada.io/permanent-id"